portfolio-project-react/src/components/Details/Details.jsx

```diff
@@ -1,12 +1,25 @@
+import { useState } from "react";
 import About from "../About/About";
 import WorkExperience from "../WorkExperience/WorkExperience";
 import styles from "./Details.module.css";
 
 const Details = ({ isEditModeEnabled }) => {
+  const [totalWorkExperience, setTotalWorkExperience] = useState([0, 0]);
```

Comment:
its better to use an object for this instead of an array with two elements `[0,0]`
because its easier to understand what is meant by `totalWorkExperience.years` and `totalWorkExperience.months`. compared to `totalWorkExperience[0]` and `totalWorkExperience[1]`, here I don't know just by reading that 0 means years and 1 means months.